src-tauri/src/main.rs

```diff
@@ -1,22 +1,100 @@
 // Prevents additional console window on Windows in release, DO NOT REMOVE!!
 #![cfg_attr(not(debug_assertions), windows_subsystem = "windows")]
+extern crate unix_named_pipe;
+extern crate ctrlc;
 
 use dotenv::dotenv;
-use std::thread;
+use std::{thread, fs, env};
+use std::path::Path;
 use tpulse::watcher::{AFKSettings, AFKWatcher};
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::sync::Arc;
+use std::thread::JoinHandle;
+use unix_named_pipe::FileFIFOExt;
+use std::io::{self, Read, BufRead};
+
+
+
 
 fn main() {
     dotenv().ok();
     env_logger::init();
 
     let afk_settings = AFKSettings::new(5000, 500);
     let afk_watcher = AFKWatcher::new(&afk_settings);
-
-    let afk_watch = thread::spawn(move || afk_watcher.run());
-
+    // let afk_watch = thread::spawn(move || afk_watcher.run());
+    let open_server_pipe = thread::spawn(move || {
+        let pipe_path = "/home/tan17112003/Desktop/myunixnamedpipe";
```

Comment:
**HARD CODE!**. Keep in mind that your computer is NOT other computers.

Reply:
I found named pipe in unix communicating with path and I put my local path in my computer.  Could you suggest any global configuration path to communicate between two computers?

Reply:
What do you mean "communicate between two computers" ? The problem here is when I pull your code, my computer cannot find the path `/home/tan17112003/Desktop/myunixnamedpipe` :) You won't tell our future users that it works on your machine, will you ?

Reply:
We need a path that can be apply for all type of systems and users. In Unix system, use absolute path like `/tmp` or create your own path like this https://github.com/TickLabVN/tpulse/blob/master/src-tauri/src/utils/get_data_directory.rs